usaspending_api/api_contracts/contracts/v1/references/filter.md

```diff
@@ -1,42 +0,0 @@
-FORMAT: 1A
```

Comment:
Are there issues with these two API Contracts deleted other than they are v1?

Since they are written we might as well leave them in-place.

Reply:
That is exactly why I took them out. Originally created them back when I reformatted all of the contracts, but I thought I would delete them so that people don't wonder why some v1 endpoints have contracts and others don't. Happy to put them back.